tests/unit-tests/src/test/java/org/apache/activemq/artemis/tests/unit/core/journal/impl/TimedBufferTest.java

```diff
@@ -205,162 +203,4 @@ public int getRemainingBytes() {
       }
 
    }
-
-   /**
-    * This test will verify if the system will switch to spin case the system can't perform sleeps timely
-    * due to proper kernel installations
-    *
-    * @throws Exception
-    */
-   @Test
```

Comment:
why did you remove this test? it was important when I added it.


I would rather keep a method public and a test than remove a test to make a method private.